age--1.1.0.sql

```diff
@@ -3198,6 +3198,14 @@ AS 'MODULE_PATHNAME';
 CREATE CAST (agtype AS text)
 WITH FUNCTION ag_catalog.agtype_to_text(agtype);
 
+CREATE FUNCTION ag_catalog.text_to_agtype(text)
+RETURNS agtype
+LANGUAGE c
+IMMUTABLE
+RETURNS NULL ON NULL INPUT
+PARALLEL SAFE
+AS 'MODULE_PATHNAME';
+
```

Comment:
You now need to add the CREATE CAST command which allows text to be cast like so `SELECT '"This is a string"'::agtype::text;` Also you need to add regression tests

Reply:
Okay, I'll add these

Reply:
Added the changes